### ISSUE TITLE
[Feature request] Follow list or Wish list

### ISSUE THREAD

**Issue by mmahmoudian** [liberapay/liberapay.com#2153]
## Summary
It would be great to be able to have a follow list or wish list.

## Justification
It happens frequently that I stumble upon some accounts that I want to keep tap on and donate to when/if I get to certain financial situation. At the moment this is done through my browser bookmark but it would be great if I can have some wish list or follow list. This also provide the following possibilities:

- being able to rotate donation between different accounts
- being able to set notification/alarm so that if the monthly donation to certain projects went below certain amount, I get notified to adjust my donation levels.

On Patreon for instance, this feature in it's simplest form exists as "Follow".

**Comment by Changaco** [comment]
This issue is a duplicate of #41 (follow list) and #48 (wish list).

**Comment by mmahmoudian** [comment]
@Changaco Thanks for pointing #41 out. It actually has such a irrelevant title imho that I didn't open it while [searching for "follow" in issues](https://github.com/liberapay/liberapay.com/issues?q=is%3Aissue+follow). but What I'm suggesting here is very far from #41, and I cannot imaging it be even remotely close to #41.

Regarding #48, it is very much the same as what I'm proposing here, except it is more bare-bone and does not have the notification that I proposed. But yes, it is the same thing.

**Comment by Changaco** [comment]
@mmahmoudian I think your notification proposal is part of #50.

I have renamed #41 to clarify what it's about.

**Comment by mmahmoudian** [comment]
> I think your notification proposal is similar to a part of #50.

Yes it is the first example there clarified it. Sorry for the duplicate. 

> I have renamed #41 to clarify what it's about.

It looks great, thanks. 👍🏼
